ng-dev/release/publish/test/test-utils/action-mocks.ts

```diff
@@ -69,6 +65,9 @@ export function setupMocksForReleaseAction<T extends boolean>(
   // to persist between tests if the sandbox git client is used.
   prepareTempDirectory();
 
+  // Set the configuration to be used throughout the spec.
+  setConfig({github: githubConfig, release: releaseConfig});
```

Comment:
Same comment on `setConfig` as above (where the Spy is actually better as it does not leak)